### ISSUE TITLE
ssl to quickfix python

### ISSUE THREAD

**Issue by k-u-s** [quickfix/quickfix#324]
Hi,

any plans on exposing SSLSocketInitiator in python?

**Comment by k-u-s** [comment]
I have made initial attempt to exposed them but i ended up with using two branches in order to do that.

On branch https://github.com/k-u-s/quickfix/tree/feature/python-ssl-swig i reverted commit a46708090444826c5f46a5dbf2ba4b069b413c58 that introduces some c++17 changes  in order to be able to use swig to generate new classes that exposes ssl.

Then i used tho classes in branch https://github.com/k-u-s/quickfix/tree/feature/python-ssl that allowed me to generate python class that were able to connect directly to exchange. With out setting up stunnel mentioned in #269 

Possibly when swig provide full support in version 4 revert mentioned on first branch would not be required in order for this procedure to work. Or if decision would be made to switch to cython from #309

System:
> PRETTY_NAME="Debian GNU/Linux 10 (buster)"
> NAME="Debian GNU/Linux"
> VERSION_ID="10"
> VERSION="10 (buster)"
> VERSION_CODENAME=buster
> ID=debian
> HOME_URL="https://www.debian.org/"
> SUPPORT_URL="https://www.debian.org/support"
> BUG_REPORT_URL="https://bugs.debian.org/"

Python:
> Python 3.7.3

Swig:
> SWIG Version 3.0.12
> Compiled with g++ [x86_64-pc-linux-gnu]
> Configured options: +pcre
> Please see http://www.swig.org for reporting bugs and further information

**Comment by MosheVai** [comment]
@k-u-s

**Comment by k-u-s** [comment]
I am not sure to which part are you referring? 

Issue was to enable python client to use SSLSocketInitiator in order to connect to fix engine that enforces ssl.
I did not find a way to connect using python client to fix engine that requires ssl without mentioned changes.

**Comment by MosheVai** [comment]
@k-u-s I was asking what did you try to do in the fix API?
I read that executing market orders requires SSL.
Does that mean that as it stands, I cant execute market orders with the quickfix python sdk unless I do the hotfix you mentioned?

**Comment by k-u-s** [comment]
I tried to to connect to DropCopy fix session.

Yes, from what i found if fix server require SSL i is not possible to connect to it via python sdk without 3rd party tools. I mange to connect and retrieve data when I set up stunnel as described in https://github.com/quickfix/quickfix/issues/269#issuecomment-629062421 but in my case I could not use that so I created this workaround.

Unless I missed something in python sdk it is not possible to connect directly to endpoint that requires SSL.
